versions/3.0.md

```diff
@@ -2287,25 +2284,32 @@ The following properties are taken directly from the JSON Schema definition and
 - minProperties
 - required
 - enum
-- type
-
-The following properties are taken from the JSON Schema definition but their definitions were adjusted to the OpenAPI Specification.
-Their definition is the same as the one from JSON Schema, only where the original definition references the JSON Schema definition, the [Schema Object](#schemaObject) definition is used instead.
-- items
-- allOf
-- oneOf
-- anyOf
-- not
-- properties
-- additionalProperties
+
+The following properties are taken from the JSON Schema definition but their definitions were adjusted to the OpenAPI Specification. 
+- type - Value MUST be a string. Multiple types via an array are not supported.
+- allOf - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
+- oneOf - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
+- anyOf - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
+- not - Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema.
+- items - Value MUST be an object and not an array. Inline or referenced schema MUST be of a [Schema Object](#schemaObject) and not a standard JSON Schema. `items` MUST be present if the `type` is `array`.
+- properties - Property definitions MUST be a [Schema Object](#schemaObject) and not a standard JSON Schema (inline or referenced).
+- additionalProperties - Value can be boolean or object. If object, definition MUST be a [Schema Object](#schemaObject) and not a standard JSON Schema (inline or referenced).
+- $ref - A [Reference Object](#referenceObject).
```

Comment:
Should we mention this at all?  To me, any place you could have a _Schema Object_ you would allow a _JSON Reference_ to a _Schema Object_ but it is an either/or, as in you cannot mix a _JSON Reference_ and a _Schema Object_ in the same object.  I think we could have some extra wording here making sure this is known to be the case, much like we document the `items` property as being applicable and required only when `type: array`.